odpf/meta/Chart.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package odpf.meta;
+
+option java_package = "io.odpf.proton.meta";
+option java_outer_classname = "Chart";
+option go_package = "github.com/odpf/proton/meta";
+
+import "odpf/meta/facets/Ownership.proto";
+import "odpf/meta/facets/Lineage.proto";
+import "odpf/meta/facets/Tags.proto";
+import "odpf/meta/facets/Custom.proto";
+import "odpf/meta/common/Timestamp.proto";
+import "odpf/meta/common/Event.proto";
+
+message ChartKey {
+  string urn = 1;
+}
+
+message Chart {
+  string urn = 1;
+  string name = 2;
+  string type = 3;
+  string source = 4;
+  string description = 5;
+  string url = 6;
+  string raw_query = 7;
```

Comment:
For Grafana, there is a possibility of more than one query. Need to consider that as well. Suggestion, we can use array of string.

Reply:
Also there is a need to know for which data source the query is run against. For example, in Grafana, a panel can have multiple queries, but the data source should only be one.

Reply:
> Also there is a need to know for which data source the query is run against. For example, in Grafana, a panel can have multiple queries, but the data source should only be one.

added `string data_source` on chart to solve this.

> For Grafana, there is a possibility of more than one query. Need to consider that as well. Suggestion, we can use array of string.

yeah agree, we need to collect more dashboard sample first though to make sure the changes would be generic.
for example, atm metabase card only has one query.

for now I would suggest adding extra queries to custom facets e.g. `raw_query_2`. Once we see a lot of chart has multiple queries then we can remodel it.

But I would assume most time-series dashboard would have multiple queries. 🤔